api/libproject/cloudsqlproxycontext.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "../../libproject/utils/common"
+require_relative "environments"
+
+class CloudSqlProxyContext
```

Comment:
This serves a completely different purpose than GcloudContext, so I pulled out the functionality. See the usage in `connect_to_cloud_db`.